Source/Core/VideoBackends/D3D/Render.cpp

```diff
@@ -930,8 +930,12 @@ void Renderer::RestoreAPIState()
   BPFunctions::SetScissor();
 }
 
-void Renderer::ApplyState(bool bUseDstAlpha)
+void Renderer::ApplyState()
 {
+  // TODO: Refactor this logic here.
```

Comment:
The refactoring would be to not use bpmem here directly. I'm already working on using some kind of rendering state objects, but as I don't want to support dual pass alpha there, this would yield a circle dependency here.